wgpu-core/src/command/compute.rs

```diff
@@ -429,7 +457,7 @@ impl Global {
             .flags
             .contains(wgt::InstanceFlags::DISCARD_HAL_LABELS);
         let hal_desc = hal::ComputePassDescriptor {
-            label: hal_label(base.label, self.instance.flags),
+            label: hal_label(base.label.as_deref(), self.instance.flags),
```

Comment:
nitpick: You introduced `….as_ref().map(|s| s.as_str())` in 2d1239a2e, then changed it to `….as_deref()` in 6fe51ff22. It took me a hot second to figure out if this was intentional or not. 😅

I've consolidated both changes into the commit introducing this code in the first place. No action needed here.